packages/@react-aria/overlays/test/calculatePosition.test.ts

```diff
@@ -216,13 +219,13 @@ describe('calculatePosition', function () {
     },
     {
       placement: 'left bottom',
-      noOffset: [50, 150, undefined, 196, 400],
-      offsetBefore: [-200, 50, undefined, 50, 500],
-      offsetAfter: [300, 350, undefined, 196, 200],
-      crossAxisOffsetPositive: [50, 160, undefined, 196, 390],
-      crossAxisOffsetNegative: [50, 140, undefined, 196, 410],
-      mainAxisOffset: [40, 150, undefined, 196, 400],
-      arrowBoundaryOffset: [50, 322, undefined, 176, 228]
+      noOffset: [50, 150, undefined, 196, 300],
+      offsetBefore: [-200, 50, undefined, 50, 200],
+      offsetAfter: [300, 350, undefined, 196, 500],
+      crossAxisOffsetPositive: [50, 160, undefined, 196, 310],
+      crossAxisOffsetNegative: [50, 140, undefined, 196, 290],
+      mainAxisOffset: [40, 150, undefined, 196, 300],
+      arrowBoundaryOffset: [50, 322, undefined, 176, 472]
```

Comment:
with the previous logic, a `left bottom` overlay had its max height calculated as if it should grow downwards but that positioning has the bottom edge of the of the overlay line up with the bottom of the trigger and so it should grow upwards.